src/main/java/strategy/SupplyOperation.java

```diff
@@ -0,0 +1,11 @@
+package strategy;
+
+import core.basesyntax.FruitTransaction;
```

Comment:
Consider placing `FruitTransaction` and `Storage` into appropriate packages to better organize the code structure, e.g., `model` for `FruitTransaction` and `db` for `Storage` as per the checklist.